docs/cloud/faq/roadmap.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Migrating to Cloud
+
+### What is the ETA on the tool being developed to assist in migration of existing customer data?
+
+We have a replication tool, which is available now. It has certain limitations, especially with performance. Get in touch, so we can help you to analyse your setup and requirements, before we can recommend using the replication tool.
+
+### Do you have indicative performance benchmarks for each of your offered cluster sizes?
+
+We are still measuring the performance of different cluster sizes and optimising the configuration for better utilisation of all the instance sizes. We will provide indicative figures on the Cloud Console as soon as we are confident in these numbers.
+
+Please remember, however, that each use case is different, and you might always get better or worse performance, compared with our advertised benchmarking figures. We can help you to analyse your needs and provide more detailed expected performance figures, please get in touch.
+
+## Features
+
+### Are there plans for the scheduled scavenge feature?
+
+Yes, we are actively working on it.
+
+### What is the ETA on the scheduled backup general availability?
+
+The scheduled backup feature is in the QA phase, and we are looking to release it soon.
```

Comment:
I think that this feature was released last night.

Reply:
let's correct this one in another PR

Reply:
I agree, we have lots of improvements in this version, and we need to get it out.